File: Remi_Hasan/src/main/java/qls/model/StyleSheet.java
package qls.model;

import qls.IQLSVisitor;

import java.util.List;

public class StyleSheet extends QLSNode {

    private final String identifier;
    private final List<Page> pages;

    public StyleSheet(String identifier, List<Page> pages) {
        this.identifier = identifier;
        this.pages = pages;
    }

    public String getIdentifier() {
        return identifier;
    }

    public List<Page> getPages() {
        return pages;
    }

    @Override
    public <T> T accept(IQLSVisitor<T> visitor) {
        return visitor.visit(this);
    }
}

File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Stylesheet.cs
using Assignment1.Model.QL.AST;
using System.Collections.Generic;
using System.Linq;

namespace Assignment1.Model.QLS.AST
{
    public class StyleSheet : ASTNode, IQLSASTNode
    {
        public IReadOnlyCollection<Page> Pages => _pages.AsReadOnly();

        private readonly List<Page> _pages;

        public StyleSheet(int lineNumber, IEnumerable<Page> pages)
        {
            _lineNumber = lineNumber;
            _pages = pages.ToList();
        }

        public void Accept(IQLSASTVisitor visitor) => visitor.Visit(this);
    }
}
